docs-kits/kits/behaviour-twin-kit/adoption-view/new-use-cases.md

```diff
@@ -0,0 +1,37 @@
+﻿---
```

Comment:
I like this section! However, does it make sense to move it rather to the development view?

Reply:
We would prefer to leave it in the Adoption View. The content referes to general steps like joining Catena-X as well as development-specific steps.